src/wallet.rs

```diff
@@ -1118,21 +1130,15 @@ pub fn verify_signature(
 
 pub fn get_cost(coin_spends: Vec<CoinSpend>) -> Result<u64, WalletError> {
     let mut alloc = Allocator::new();
+    let constants = TargetNetwork::Mainnet.get_constants();
```

Comment:
I assume testnet11/mainnet will have the same relevant constant ts

Reply:
idk I didn't change it